backend/fms_core/tests/test_models.py

```diff
@@ -291,41 +287,10 @@ def test_sample_kind(self):
                 self.assertIn('tissue_source', e.message_dict)
                 raise e
 
-    def test_volume_used(self):
-        # volume_used cannot be None for an extracted_sample
-        volume_used = None
-        invalid_volume_used = Sample(**create_extracted_sample(sample_kind=self.sample_kind_DNA, volume_used=volume_used,
-                                                               **self.constants))
-        invalid_volume_used.save()
-        p = Process.objects.create(protocol=self.extraction_protocol, comment="Process test_volume_used")
-        with self.assertRaises(ValidationError):
-            try:
-                ps = ProcessSample.objects.create(process=p,
-                                                  source_sample=self.parent_sample,
-                                                  execution_date=timezone.now(),
-                                                  volume_used=volume_used,
-                                                  comment="ProcessSample test_volume_used")
-                SampleLineage.objects.create(parent=self.parent_sample, child=invalid_volume_used, process_sample=ps)
```

Comment:
You may want to keep the first part of this test which is still valid given that you need a ProcessSample.volume_used for extraction

Reply:
yes but I think that is not where it should be tested. this is Sample Model test, and volume_used is no longer a Sample attr. If there is a need for test it should be made on ProcessSample

Reply:
This is actually an extraction test as far as i can see ... as you mention the test is performed on ProcessSample (in the try). The test could also be moved to a ProcessSample specific section it would be logical, but i would not drop the test altogether...